lucida/weather/server/WeatherServer.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import sys
+sys.path.append('../')
+
+from lucidatypes.ttypes import QuerySpec
+from lucidaservice import LucidaService
+
+from thrift.transport import TSocket
+from thrift.transport import TTransport
+from thrift.protocol import TBinaryProtocol
+from thrift.server import TServer
+
+import urllib2
+import json
+import urllib
+
+port = 8088
+owm_api_key = "362537b891e6df03a316e82565fe4df3"
```

Comment:
ew Magic Numbers, can you move this to a config and put in the README that it needs to be set or something? Should be a "secret"